examples/bert/bert_ptq_cpu.json

```diff
@@ -4,7 +4,10 @@
         "config": {
             "hf_config": {
                 "model_name": "Intel/bert-base-uncased-mrpc",
-                "task": "text-classification"
+                "task": "text-classification",
```

Comment:
Let us add this for cuda case. For SdpaAttention, onnxruntime run failed when enable_cuda_graph

Reply:
So long as the CI could succeeds, I prefer not add this kinds of limitation.